os/kernel/log_dump/log_dump.c

```diff
@@ -47,11 +54,15 @@
 #define MAX(a, b)		   (((a) > (b)) ? (a) : (b))
 #endif
 
-#define LOG_DUMP_OK			 0
+#define LOG_DUMP_OK			0
+#define LOG_DUMP_OK_HEAPCHK		1
```

Comment:
Could you add the description for this?
It is also OK, so it is better to explain which is different from `LOG_DUMP_OK`.